ding/envs/env_wrappers/env_wrappers.py

```diff
@@ -273,26 +288,37 @@ def reward(self, reward):
 class ActionRepeatWrapper(gym.Wrapper):
     """
     Overview:
-        Repeat the action to step with env.
-    Interface:
-        ``__init__``, ``step``
+        The ActionRepeatWrapper class is a gym wrapper that repeats the same action for a number of steps.
```

Comment:
add more explanation for when to use this wrapper

Reply:
polished